modules/tensor_mechanics/test/tests/j_integral/tests

```diff
@@ -26,6 +26,26 @@
     requirement = 'The domain integral action shall error out if the blocks selected do not have the '
                   'adequate material objects.'
   []
+  [j_2d_output_vpp]
+    type = 'CheckFiles'
+    issues = '#25164'
+    input = 'j_integral_2d.i'
+    check_files = 'j_integral_2d_output_vpp_out.csv'
+    check_not_exists = 'j_integral_2d_output_vpp_out_*VPP*.csv'
+    cli_args = "DomainIntegral/output_vpp=false Outputs/file_base=j_integral_2d_output_vpp_out"
+    requirement = 'The domain integral action shall suppress the vector postprossesor output '
+                  'and stop generating a csv file for each integration ring at each time step '
+                  'when csv output is turned on.'
```

Comment:
'when csv output is turned on' means general csv output? Intuitively it may make more sense to say something like 'when the flag output_vpp is chosen to be false'.